clis/daemon-common/src/daemon.rs

```diff
@@ -104,11 +109,11 @@ impl Daemon {
 /// # Returns
 ///
 /// Result containing the path of the daemons work directory.
-fn get_wd_path() -> Result<PathBuf> {
+fn get_wd_path(daemon_name: &str) -> Result<PathBuf> {
```

Comment:
This looks strange to me as per generic `teliod`, we are getting current path to "client" binary, and later expecting to get something with the `*.sock` as unix socket. And later adding `/var/run/xxx` here? Or am I missing something?

Reply:
Yeah, it does look strange to me too, it was some idea to make a private helper function for a public interface.

Reply:
Maybe lets us `/run` :thinking: as per FHS standard https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s15.html and add the fallback to `/var/run` just in case (e.g. if `/run` does not exist)

> System programs that maintain transient UNIX-domain sockets must place them in this directory or an appropriate subdirectory as outlined above.

This should also be compatible with MacOS as well :crossed_fingers:.